artbutton.js

```diff
@@ -5,6 +5,11 @@
  * @namespace ShowArt
  */
 class ShowArt {
+	/** 
+	 * cache of module settings
+	 */ 
+	static _cachedEnableHudButton = true;
```

Comment:
Rather than a "cache" of settings, I would prefer to add a getter that actually grabs the settings from Foundry every time. It's preferable to not have multiple copies of the same information that have to be kept in sync.